swag/artefacts/accounts.py

```diff
@@ -43,28 +42,32 @@ def delattr(self, __name: str) -> None:
 class Account:
     swag_balance: Swag = attrib(init=False, default=Swag(0))
     style_balance: Style = attrib(init=False, default=Style(0))
-    yfu_wallet: List[Yfu] = attrib(init=False, default=list())
+    yfu_wallet: Set[YfuId] = attrib(init=False, factory=set)
 
-    def __iadd__(self, value: Union[Swag, Style]):
+    def __iadd__(self, value: Union[Swag, Style, YfuId]):
         if type(value) is Swag:
             self.swag_balance += value
         elif type(value) is Style:
             self.style_balance += value
+        elif type(value) is YfuId:
+            self.yfu_wallet.add(value)
         else:
             raise TypeError(
-                "Amounts added to SwagAccount should be either Swag or Style."
+                "Amounts added to SwagAccount should be either Swag, Style or YfuId."
             )
         return self
 
-    def __isub__(self, value: Union[Swag, Style]):
+    def __isub__(self, value: Union[Swag, Style, YfuId]):
         try:
             if type(value) is Swag:
                 self.swag_balance -= value
             elif type(value) is Style:
                 self.style_balance -= value
+            elif type(value) is YfuId:
+                self.yfu_wallet.remove(value)
             else:
                 raise TypeError(
-                    "Amounts subtracted to SwagAccount should be either Swag or Style."
+                    "Amounts subtracted to SwagAccount should be either Swag, Style or YfuId."
```

Comment:
But, yeah, okey I guess?

Reply:
Yep, as you said, it's just for have a generic Transaction code, because why coding exactly the same thing twice ?

Ok here we don't work with amount of value, but just id directly, but it's tottaly wrapped into the new definition of __add__ and __sub__ concerning yfus